src/mailer/config.ts

```diff
@@ -0,0 +1,84 @@
+import * as t from "io-ts";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { MailMultiTransportConnectionsFromString } from "./multi_transport_connection";
+
+// exclude a specific value from a type
+// as strict equality is performed, allowed input types are constrained to be values not references (object, arrays, etc)
+// tslint:disable-next-line max-union-size
+export const AnyBut = <A extends string | number | boolean | symbol, O = A>(
```

Comment:
this belongs to io-ts-commons, but at least, can we move it into ustils/types.ts ?

Reply:
I'd rather not do it because I didn't stress `AnyBut` enough to make it a general-purpose utility. It is performing good for this specific case, but I'm afraid it might not be the same in other contexts.